### PR TITLE
chore(android/samples,oem/fv/android) Update dependencies

### PR DESCRIPTION
**Pull request by darcywong00**
Follow-on to #5098 and #5126

This updates the dependencies to the Android Sample apps and FV Android app.
* Also update targetSDKVersion to 30

### CODE REVIEW

File: android/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Prerequisites
 * Android Studio 4.1+
 * Java SE Development Kit 8 
-* [Node.js](https://nodejs.org/) 8.9+ (for building KeymanWeb)
+* [Node.js](https://nodejs.org/) 14.17+ (for building KeymanWeb)
```
Comment (mcdurdin) [inline android/README.md:6]: Is this strictly correct? @jahorton

**Review by mcdurdin — APPROVED** [review]

File: android/README.md (hunk 1 above)
Comment (darcywong00) [reply android/README.md:6]: fwiw, I updated my development machine to node 16.20 in order to get npm 7 (discussed in #5092)

Comment (jahorton) [reply android/README.md:6]: FWIW:

Screen Shot 2021-05-21 at 1 07 27 PM

Comment (jahorton) [reply android/README.md:6]: Also, NPM 7 shouldn't be a hard requirement.  Its new package lockfile format is designed to be backward-compatible with NPM 5 and 6, though they won't be able to process some of its enhancements.  I put a link in #5092 about that.

Comment (darcywong00) [reply android/README.md:6]: From https://nodejs.org/en/about/releases/
I'm fine keeping the current 14 LTS.

Node.js 12 ceases to be maintained Apr 2022.

While the current package lockfile is backward-compatible with NPM 5 and 6, might as well have all developers use NPM 7 so we don't have to wrangle with package lockfile diffs.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.52-alpha](https://keyman.com/downloads/releases/alpha/15.0.52)

**Pull request merged**